main.go

```diff
@@ -204,6 +205,19 @@ func clientAuthenticate(c *client.ApiClient, storeFile string) client.AuthToken
 	}
 }
 
+func stressTestClientServerIterator() func() *client.MenderServer {
+	serverIteratorFlipper := true
+	return func() *client.MenderServer {
+		// Only empty the iterator after five attempts
```

Comment:
Does the comment match what actually happens? Looks like it's binary flipping.

Reply:
Nice, thank you!